### PR TITLE
Trader Adjustments

### PR DESCRIPTION
**Pull request by Owai-Seek**
## About The Pull Request

- Organized sold food file.
- Adjust farming apprentice prices.
- Adjust basic mining kit price.
- Adjust atmos trader prices.
- Added space suit to atmos trader.
- Food Traders will buy (some) ingredients or products.
- Fixed a tiny error with atmos vendor. (wrong variable?)

Tested it a few times last night, everything -should- be working.

## Why It's Good For The Game

Adds a way to get space suits. Also allows the chef and botanist to contribute to Bearcat Economy

## Changelog
:cl:
add: Food traders will buy select goods.
add: Atmos traders now carry a space suit.
balance: Adjusted multiple trader prices.
/:cl:

### CODE REVIEW

**Comment by Azarak** [comment]
I'd say lower the amounts of the items on most things and up their price by a bit. It's not fun to sell things 3 credits each.

**Comment by Owai-Seek** [comment]
Well, the idea with a lot of the food vendors was that these items can be mass produced, so selling a bunch of them for low amounts would be expected. (Especially since you can stick them in plant bags/crates and send them down a floor.)

I can probably adjust stuff tomorrow once I get up, I forgot that vendors can have a bit of variance.

**Comment by Azarak** [comment]
> 
> 
> Well, the idea with a lot of the food vendors was that these items can be mass produced, so selling a bunch of them for low amounts would be expected. (Especially since you can stick them in plant bags/crates and send them down a floor.)
> 
> I can probably adjust stuff tomorrow once I get up, I forgot that vendors can have a bit of variance.

The idea behind the stock limit is that it prevents the benefit of setting mass productions up, so you can have sane prices on things, because their export is gonna be limited arbitraily

File: code/modules/trading/bought_goods/food.dm [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	name = "eggs"
+	cost = 10
+	trading_types = list(/obj/item/food/egg = TRADER_THIS_TYPE)
+	stock_high = 18
+	stock_low = 8
+
+/datum/bought_goods/produce // 80-160c
+	name = "grown produce"
+	cost = 10
+	trading_types = list(/obj/item/food/grown/ = TRADER_THIS_TYPE)
```
Comment (Azarak) [inline code/modules/trading/bought_goods/food.dm:38]:
```suggestion
	trading_types = list(/obj/item/food/grown = TRADER_SUBTYPES)
```

**Review by Azarak — APPROVED** [review]

**Pull request merged**